mxcube3/core/components/workflow.py

```diff
@@ -25,7 +25,7 @@ def get_available_workflows(self):
         except Exception:
             pass
 
-        if hasattr(beamline, "gphl_workflow"):
+        if getattr(beamline, "gphl_workflow", None):
```

Comment:
Nothing wrong with this, but it should not be necessary. What error happens if you use 'hasattr'?

Reply:
There is a subtle difference, `hasattr` will return `True` if the attribute exists but it can still have the value `None`
Meaning that the expression` beamline.gphl_workflow.get_available_workflows() `will fail. On the other hand
`getattr` with default None will return `False` and the statement never executed.